### DISCUSSION TITLE
cant see any components

### DISCUSSION THREAD

**Discussion by gerofeldmann** [SukramJ/aiohomematic#959] [Q&A]
Hello,
I installed the addon and after installation I configured IP etc., opened the ports in Homematic. But still I do not get to see any components.
The log says that the is no connection.....

who can help?

**Comment by SukramJ** [comment]
Not much can be done with this information. Please create an [issue](https://github.com/danielperna84/hahomematic/issues) and add as much information as possible. Esp. logs are relevant.
